### PR TITLE
Explore courses button padding fixed

### PR DESCRIPTION
**Pull request by muhammedanaskhan**

### CODE REVIEW

File: Components/HeroSection.tsx [hunk 1]
```diff
@@ -78,7 +78,8 @@ const HeroSection = () => {
               >
                 <span className="inline-flex bg-black "></span>
 
-                <h4 className="text-xl  font-archivo dark:text-zinc-800 ">Explore Courses</h4>
+                <h4 className="text-xl  font-archivo">Explore Courses</h4>
```
Comment (lalitmee) [inline Components/HeroSection.tsx:81]: please remove the extra space in between

File: Components/HeroSection.tsx [hunk 2]
```diff
@@ -78,7 +78,8 @@ const HeroSection = () => {
...
+                <span className="inline-flex bg-black "></span>
```
Comment (lalitmee) [inline Components/HeroSection.tsx:82]: please remove the extra space before `"`

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by lalitmee** [comment]
@muhammedanaskhan, can you please update your PR description with a screenshot of the before and after changes?

**Comment by PiyushKalyanpy** [comment]
@muhammedanaskhan please add the issue number to the description

**Comment by PiyushKalyanpy** [comment]
The requested changes have not been implemented, so we are closing this pull request. Please feel free to resubmit the changes when they are ready.

**Pull request closed without merging**